sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/WritableColumnVector.java

```diff
@@ -324,24 +315,27 @@ public void putDecimal(int rowId, Decimal value, int precision) {
   @Override
   public UTF8String getUTF8String(int rowId) {
```

Comment:
Shall we add comment that `getUTF8String` reuse the data in column vector? It seems different than other getXXX APIs

Reply:
+1